tests/opentracer/test_tracer_gevent.py

```diff
@@ -1,67 +1,74 @@
-import pytest
 import gevent
-
+import pytest
+import unittest
+from nose.tools import eq_
 from opentracing.scope_managers.gevent import GeventScopeManager
+
+import ddtrace
+from ddtrace.contrib.gevent import patch, unpatch
+from ddtrace.opentracer.utils import get_context_provider_for_scope_manager
+
 from tests.opentracer.test_tracer import get_dummy_ot_tracer
 
 
 def get_dummy_gevent_tracer():
-    return get_dummy_ot_tracer('gevent', {}, GeventScopeManager())
+    from ddtrace.contrib.gevent import context_provider
```

Comment:
Fixtures also here. But I don't think we need a `get_dummy_*_tracer()`. We can make it configurable using `pytest` fixtures. I'd say to refactor a bit this testing part because it's creating a level of indirection.